File: LeetCode_Solutions/Java/80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
            if (nums.length == 0) return 0;
    int j = 0;
    int count = 1;
    for (int i = 1; i < nums.length; i++) {
      if (nums[i - 1] == nums[i]) {
        count++;
      } else {
        count = 1;
      }
      if (count == 1 || count == 2) {
        j++;
        nums[j] = nums[i];
      }
    }
    return j + 1;
    }
}

File: LeetCode_Solutions/Java/80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0

        for num in nums:
            if i < 2 or num > nums[i - 2]:
                nums[i] = num
                i += 1

        return i

#        for num in nums:
#            while nums.count(num)>2:
#                nums.remove(num)
#        return len(nums)